src/main/java/mate/academy/springboot/swagger/mapper/Mapper.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.springboot.swagger.mapper;
+
+import java.util.List;
+import mate.academy.springboot.swagger.dto.ProductDto;
+import mate.academy.springboot.swagger.model.Product;
+
+@org.mapstruct.Mapper(componentModel = "spring")
```

Comment:
Think about how to import. Can you give a different name to the class?

Reply:
done